10up/classifai
# Add Azure Language Services provider

### Description of the Change

This PR adds Azure Language Services as a provider and adds it to the Excerpt Generation feature.

Closes #159 

### How to test the Change

Testing Steps:
1. Create new Azure resource for Language Services
2. Enter Credentials into the Excerpt Generation feature screen
3. Confirm valid credentials allow feature to be enabled
4. Test 'Generate Excerpt' tag for allowed post types.

### Changelog Entry

> Added - Azure Language Services provider with Excerpt Generation.

### Credits

Props @psorensen @cadic @jeffpaul @Sidsector9 @dkotter 

### Checklist:
